tests/output/printers/LogPrinterTest.py

```diff
@@ -66,12 +65,8 @@ def test_set_state(self):
         uut = LogPrinter()
         state = uut.__getstate__()
         uut.__setstate__(state)
-        self.assertEqual(uut.logger, logging.getLogger('coala.raw'))
-
-    def test_colored_printer(self):
-        uut = LogPrinter(printer=ColorPrinter())
-        self.assertIs(uut.logger, logging.getLogger('coala.colored'))
+        self.assertEqual(uut.logger, logging.getLogger('coalib'))
```

Comment:
Why would this be `self.assertEqual` but the other be `self.assertIs`? IIUC both should be `self.assertIs`?